### PR TITLE
fix: default log level to debug

### LINKED ISSUE #88: log('debug', 'useful message') does not works on cozy instance in debug mode
After passing a cozy instance to debug mode, by running the following command:

```shell
cozy-stack instances debug <domain> true
```

The instruction `log('debug', 'useful message') does not output anything.

Expected behavior:

debug message is shown

### PR DESCRIPTION
**Pull request by doubleface**
Since the stack already handle logs filtering itself.
Fixes #88

### CODE REVIEW

File: libs/log-filters.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const levels = {
-  debug: 0,
   secret: 0,
+  debug: 10,
```
Comment (ptbrowne) [inline libs/log-filters.js]: shouldn't other levels be increased by 10 ?

**Comment by ptbrowne** [comment]
small question otherwise lgtm :)

File: libs/log-filters.js (hunk 1 above)
Comment (doubleface) [reply libs/log-filters.js]: Right

File: libs/logger.js [hunk 2]
```diff
@@ -2,10 +2,10 @@
 const { env2formats } = require('./log-formats')
 const { filterLevel, filterSecrets } = require('./log-filters')
 
-const { DEBUG, NODE_ENV, LOG_LEVEL } = process.env
+const { NODE_ENV, LOG_LEVEL } = process.env
 const env = (env2formats[NODE_ENV] && NODE_ENV) || 'production'
 
-let level = LOG_LEVEL || (DEBUG && DEBUG.length ? 'debug' : 'info')
+let level = LOG_LEVEL || 'debug'
```
Comment (gregorylegarec) [inline libs/logger.js]: Does that mean that by default, every konnector is at `debug` level, and the log filtering is done by the stack ?

**Comment by doubleface** [comment]
That's it

File: libs/logger.js (hunk 2 above)
Comment (doubleface) [reply libs/logger.js]: That's it

**Pull request merged**